### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'id')

### ISSUE THREAD

**Issue by koblack** [Novik/ruTorrent#2597]
### Please complete the following tasks.

- [X] Web browser cache cleared
- [X] Copied everything if upgrading to v4.0 or master
- [X] Web browser and ruTorrent version provided

### Tell us about your environment

Web Browser: Google Chrome 119.0.6045.124
ruTorrent branch v4.2: https://github.com/Novik/ruTorrent/commit/d8dc2956b0e3faab6b9a13fa9e4b008b91894e51
PHP: PHP 7.4.3
OS: Ubuntu 20.04.6 LTS
web server: lighttpd/1.4.55

### What were you trying to do?

1. search and press enter
2. remove tag
3. select torrant and you get error

![id-error](https://github.com/Novik/ruTorrent/assets/25719620/c4507685-63ea-402d-9c30-0a296061fe43)

### What us the error message?

Uncaught TypeError: Cannot read properties of null (reading 'id')

### Give us the stack trace.

``` 
Uncaught TypeError: Cannot read properties of null (reading 'id') from TypeError: Cannot read properties of null (reading 'id')
    at theWebUI.updateLabel (<anonymous>:7030:19)
    at Object.updateLabels (http://box/ruTorrent/js/webui.js?v=428:2355:10)
    at theWebUI.updateLabels (<anonymous>:7047:22)
    at theWebUI.rebuildTrackersLabels (<anonymous>:7131:7)
    at theWebUI.addTrackers (<anonymous>:6971:12)
    at Object.<anonymous> (http://box/ruTorrent/js/rtorrent.js?v=428:1301:22)
    at c (http://box/ruTorrent/js/jquery.js?v=428:2:25266)
    at Object.fireWith [as resolveWith] (http://box/ruTorrent/js/jquery.js?v=428:2:26015)
    at l (http://box/ruTorrent/js/jquery.js?v=428:2:77721)
    at XMLHttpRequest.<anonymous> (http://box/ruTorrent/js/jquery.js?v=428:2:80204)
```

**Comment by stickz** [comment]
I was able to reproduce this error with **Show label size in Search panel** enabling in the formatting options.

**Comment by stickz** [comment]
@koblack Could you confirm that https://github.com/Novik/ruTorrent/commit/270c53c070737790a6b5572836d7de0cfe3c19ae resolves the problem?

**Comment by koblack** [comment]
Partially...
"Remove tag" option now works
"Remove all tags" still doesn't work

**Comment by stickz** [comment]
@koblack Could you confirm a complete fix with https://github.com/Novik/ruTorrent/commit/760f58bde8c7eba3a4a5f0ce5ce8f4a16c3d422f ?

**Comment by stickz** [comment]
Fixed in [v4.2.9](https://github.com/Novik/ruTorrent/releases/tag/v4.2.9).
